### DISCUSSION TITLE
2 different Tuya TRV valves TS0601: One supported, the other one unsupportted

### DISCUSSION THREAD

**Discussion by agak79** [Koenkk/zigbee2mqtt#21447] [Devices]
Hello

I bought 2 tuya TRV valves and tried to configured them in HA zigbee2Mqqt: The first one worked and was recognized as TA0601_thermostat, but the second one didn't worked and is detected as unsupported device.

First one dev console:
`const definition = {
    zigbeeModel: ['TS0601'],
    model: 'TS0601',
    vendor: '_TZE200_cpmgn2cf',
    description: 'Automatically generated definition',
    extend: [],
    meta: {},
};

module.exports = definition;`

Second one: 

`
const definition = {
    zigbeeModel: ['TS0601'],
    model: 'TS0601',
    vendor: '_TZE200_znlqjmih',
    description: 'Automatically generated definition',
    extend: [],
    meta: {},
};`

Is any way to force the 2nd to be recognized as the first?

**Comment by Koenkk** [comment]
`_TZE200_cpmgn2cf` will be supported in the 1 March z2m release.
